data/journal_check/bug_refs.json

```diff
@@ -480,5 +480,19 @@
             "sle-micro": ["5.4" , "5.5"]
         },
         "type": "bug"
+    },
+    "bsc#1215378": {
+        "description": "libvirtd.*: error from service: GDBus.Error:org.freedesktop.login1.OperationInProgress: The operation inhibition has been requested for is already running",
+        "products": {
+            "sle-micro": ["5.4" , "5.5"]
+        },
+        "type": "bug"
+    },
+    "bsc#1215405": {
+        "description": "SELinux is preventing.* from.* access.*|setroubleshoot.*: failed to retrieve rpm info for path.*",
```

Comment:
I am not sure about this one, we might be hide future messages...
Maybe something like:
```
"setroubleshoot.*SELinux is preventing (.*rebootmgrd|.*NetworkManager|sshd).* from.* access.*|setroubleshoot.*: failed to retrieve rpm info for path.*"
```
No need to run all VRs again.